src/vs/workbench/services/progress/test/browser/progressIndicator.test.ts

```diff
@@ -122,7 +123,8 @@ suite('Progress Indicator', () => {
 		let testProgressBar = new TestProgressBar();
 		let viewletService = new TestViewletService();
 		let panelService = new TestPanelService();
-		let service = new CompositeProgressIndicator((<any>testProgressBar), 'test.scopeId', true, viewletService, panelService);
+		let viewsService = new TestViewsService();
```

Comment:
@eamodio maybe include a test for the view change to be triggering progress show/hide?

Reply:
Added a couple of checks to the existing tests.